benchmark/syclblas/blas1/iamax.cpp

```diff
@@ -46,21 +46,21 @@ void run(benchmark::State& state, ExecutorType* executorPtr, index_t size,
 
   // Create data
   std::vector<scalar_t> v1 = blas_benchmark::utils::random_data<scalar_t>(size);
-  blas::IndexValueTuple<scalar_t, index_t> out(-1, -1);
+  blas::IndexValueTuple<index_t, scalar_t> out(-1, 0);
```

Comment:
Is there a way to create this from the constants without including the operators hpp file? The following is a bit verbose:
```
auto out = constant_pair<IndexValueTuple<index_t, scalar_t>, const_val::m_one, const_val::zero>::value();
```

I thought we would also have something simple like:
```
auto out = constant<IndexValueTuple<index_t, scalar_t>, const_val::amin>::value();
```

Reply:
No no need. The constant and operator is pure internal code. So no need to use it in user code.